management/cmd/management.go

```diff
@@ -178,8 +179,13 @@ var (
 				tlsEnabled = true
 			}
 
-			httpAPIHandler, err := httpapi.APIHandler(accountManager, config.HttpConfig.AuthIssuer,
-				config.HttpConfig.AuthAudience, config.HttpConfig.AuthKeysLocation, appMetrics)
+			httpApiAuthCfg := httpapi.AuthCfg{
+				Issuer:       config.HttpConfig.AuthIssuer,
+				Audience:     config.HttpConfig.AuthAudience,
+				UserIDClaim:  config.HttpConfig.AuthUserIDClaim,
```

Comment:
This value will be empty on upgrade as it didn't exist before. We can add a small check and set it to "sub"

Reply:
This property initialized in the Claims Extractor, and if UserIDClaim is empty, we use `sub`.
https://github.com/gigovich/netbird/blob/feat-add-custom-id-claim/management/server/jwtclaims/extractor.go#L62-L62